pandas/core/arrays/categorical.py

```diff
@@ -2231,33 +2231,32 @@ def unique(self):
         -------
         unique values : ``Categorical``
 
+        See Also
+        --------
+        unique
```

Comment:
```suggestion
        pandas.unique
```

(I *think* it will otherwise refer to itself, as i assume the local namespace has precendence on the top-level one)